### ISSUE TITLE
How to get Video embeds?

### ISSUE THREAD

**Issue by spences10** [ChristopherBiscardi/christopherbiscardi.github.com#92]
Hey Chris,

How do you implement this:

https://github.com/ChristopherBiscardi/christopherbiscardi.github.com/blob/c3d414dbd8631758461fbe7aea088c0781ec743a/gatsby-config.js#L42-L48

Trying something similar (copypasta actually) and getting errors because React is not in scope??

**Comment by ChristopherBiscardi** [comment]
@spences10 did you get this worked out? I remember we had a short conversation about it on Twitter

**Comment by ChristopherBiscardi** [comment]
Also, you probably need `import React from 'react'` at the top of your `globalScope`. I use `gatsby-plugin-emotion` here which changes my `createElement` implementation, so I don't actually need React in scope for mine.

**Comment by spences10** [comment]
Hey @ChristopherBiscardi, I used @kentcdodds [remark-embedder] plugin in the end as it solved the twitter embed issue I was having as well.

But for clarity when you say `globalScope` what is it that you mean? I thought that I would need to include something with `wrapRootElement`?

Closing this now.

[remark-embedder]: https://github.com/kentcdodds/kentcdodds.com/tree/master/plugins/remark-embedder
